prboom2/src/MUSIC/alsaplayer.h

```diff
@@ -0,0 +1,61 @@
+/* Emacs style mode select   -*- C++ -*-
+ *-----------------------------------------------------------------------------
+ *
+ *
+ *  PrBoom: a Doom port merged with LxDoom and LSDLDoom
+ *  based on BOOM, a modified and improved DOOM engine
+ *
+ *  Copyright (C) 2011 by
```

Comment:
Please add your copyright here.

Reply:
The original code was the Portmidi player, which it seems was written by Nicholai Main. Is it fine if I keep his name in there?